ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+[Before submitting an issue, please check that your isssue hasn't been already filed]
+
+### Description
+
+[Description of the issue]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### Expected behavior
+
+[What you expect to happen]
+
+### Actual behavior
+
+[What actually happens]
+
+### Reproducibility
+
+[What percentage of the time does it reproduce?]  
+
+### Versions
+
+[If a tagged version, you can get this information by inspecting the `ITK_VERSION_MAJOR` `ITK_VERSION_MINOR` and `ITK_VERSION_PATCH` variable values in the `ITKConfig.cmake` file]
+
+[If the commit number is required, run `$ git rev-parse --short HEAD`]
+
+### Platform
+
+[Which your OS and compiler are]
+
+
+### Additional Information
+
+[Any additional information, configuration or data that might be necessary to reproduce the issue.]
+
+### Label
+
+[Choose the label that best fits the issue]
```

Comment:
Give a list of labels here to choose from.  These labels should match the GITHUB labels for issues.

Reply:
Or at least give a URL of where to find the labels.

Reply:
@hjmjohnson thanks for the review ! This was just a proposal; the question is whether we will effectively be using labels. The VTK folks do have some, and they do use them frequently. If for any reason it is decided that we will not be using labels (despite I'd be for proposing/using them), then adding such a section is not worthwhile.

Reply:
Maybe we can add the *Label* and *Milestones* section later once we have figured out what labels to use and sorted out the milestoning system.